### PR TITLE
Fields order

### PR DESCRIPTION
**Pull request by folix-01**
Maintain the form field order in the email message + fields spacing

### CODE REVIEW

**Comment by cekk** [comment]
@folix-01 fix ci please

**Review by mamico — APPROVED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -4,6 +4,9 @@ Changelog
 3.1.1 (unreleased)
 ------------------
 
+- Maintain the fields order in email message.
```
Comment (cekk) [inline CHANGES.rst:7]: that's not the only change that you've done

**Review by cekk — CHANGES_REQUESTED** [review]

**Comment by folix-01** [comment]
based on https://github.com/collective/collective.volto.formsupport/pull/65

File: src/collective/volto/formsupport/profiles/default/metadata.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metadata>
-  <version>1300</version>
+  <version>1301</version>
   <dependencies>
-    <!--<dependency>profile-plone.app.dexterity:default</dependency>-->
+    <dependency>profile-collective.volto.otp:default</dependency>
```
Comment (mamico) [inline src/collective/volto/formsupport/profiles/default/metadata.xml:5]: this is not related to this PR.

Comment (folix-01) [reply src/collective/volto/formsupport/profiles/default/metadata.xml:5]: https://github.com/collective/collective.volto.formsupport/pull/64#issuecomment-2263197498

**Review by cekk — APPROVED** [review]
